spec/support/rails_template.rb

```diff
@@ -97,6 +101,12 @@ def set_id
 
 generate 'active_admin:install'
 
+inject_into_file 'app/models/admin_user.rb', %q{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%q` only for strings that contain both single quotes and double quotes.